src/app/swap/[[...path]]/path-resolver.ts

```diff
@@ -16,46 +21,45 @@ export class PathResolver {
   // Returns resolved token path - always.
   // If the path shouldn't be resolvable - a default will be returned.
   resolve(path: string[]): ResolvedTokenPath {
+    const defaultIndex = getDefaultIndex()
     const symbols = this.convertPathToSymbols(path)
-    /// both null
+
     if (symbols.inputToken === null && symbols.outputToken === null)
       return {
+        isMinting: true,
         inputToken: ETH,
-        outputToken: indexNames[0],
+        outputToken: defaultIndex,
       }
 
-    if (symbols.inputToken === null) {
-      const outputToken = this.resolveToken(symbols.outputToken!)
-      const isIndex = indexNames.find(
-        (token) => token.symbol === outputToken.symbol
-      )
+    let inputToken = this.resolveToken(symbols.inputToken ?? ETH.symbol)
+    let outputToken = this.resolveToken(
+      symbols.outputToken ?? defaultIndex.symbol
+    )
+    const inputTokenIsIndex = indicesTokenList.some(
+      (token) => token.symbol === inputToken.symbol
+    )
+    const outputTokenIsIndex = indicesTokenList.some(
+      (token) => token.symbol === outputToken.symbol
+    )
```

Comment:
Do we need to worry about casing at all?

Reply:
This is basically handled in `resolveToken` which returns objects we control, so symbols should always have matching formats. https://github.com/IndexCoop/index-app/blob/4d0e35051d5aa66e9932ea653bc8b2a65966634a/src/app/swap/%5B%5B...path%5D%5D/path-resolver.ts#L84